zproject/local_settings_template.py

```diff
@@ -45,27 +45,30 @@
 # SSO_APPEND_DOMAIN = "example.com")
 SSO_APPEND_DOMAIN = None
 
-# Configure the outgoing SMTP server below. The default configuration
-# is prepopulated for GMail servers.  Change EMAIL_HOST for other
-# hosts, or leave it unset or empty to skip sending email.  Note if
-# you are using a GMail account to send outgoing email, you will
-# likely need to configure that account as "less secure" here:
-# https://support.google.com/accounts/answer/6010255.
+# Configure the outgoing SMTP server below. You will need to complete
+# the following steps:
```

Comment:
This may be overkill, but it would be useful for me to see a list of the categories of outgoing emails that are sent (missed messages, are there others?) so as to know why it's important to configure this (side question - is outgoing SMTP actually _required_ for a self-hosted Zulip install or is it only required if they want to support missed messages & whatever the other emails are)?